src/main/java/dev/jbang/source/Project.java

```diff
@@ -164,6 +164,31 @@ public Project setJavaVersion(String javaVersion) {
 		return this;
 	}
 
+	@Nonnull
+	public String getFullJavaVersion() {
+		String fullJavaVersion = getJavaVersion();
+		if (fullJavaVersion == null) {
```

Comment:
really not a fan of this reliance on sdkman when jbang doesn't even use/enable sdkman jdks - exactly because there is no reliable way to query/match java versions.